src/main/scala/uk/gov/ons/bi/dataload/utils/FixSic.scala

```diff
@@ -0,0 +1,69 @@
+package uk.gov.ons.bi.dataload.utils
+
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.functions.udf
+
+object FixSic {
+
+  lazy val spark: SparkSession = {
+    SparkSession
+      .builder()
+      .master("local")
+      .appName("spark tests")
+      .getOrCreate()
+  }
+
+  val colNames = Seq(
+    "SICCode.SicText_1",
+    "SICCode.SicText_2",
+    "SICCode.SicText_3",
+    "SICCode.SicText_4"
+  )
+
+  val validNonTradingSic = Seq(
+    "74990",
+    "98000",
+    "99000",
+    "99999"
+  )
+
+  def createValidSicList(filepath: String): Seq[String] = {
+
+    val sicCodeIndex = spark.read.option("header", "true").csv(filepath)
+    val sic07 = sicCodeIndex.select("SIC 2007").collect().map(x => x.toString()).toSeq.map(x => x.replaceAll("[\\[\\]]",""))
+    sic07
```

Comment:
You don't need to make `sic07` a `val`, just leave `sicCodeIndex.select("SIC 2007").collect().map(x => x.toString()).toSeq.map(x => x.replaceAll("[\\[\\]]",""))` at the bottom of the method and it will be returned.